packages/core/src/server-runtime-client.ts

```diff
@@ -46,7 +49,7 @@ export class ServerRuntimeClient<
    * @inheritDoc
    */
   public eventFromException(exception: unknown, hint?: EventHint): PromiseLike<Event> {
-    return Promise.resolve(eventFromUnknownInput(getCurrentHub, this._options.stackParser, exception, hint));
+    return resolvedSyncPromise(eventFromUnknownInput(getCurrentHub, this._options.stackParser, exception, hint));
```

Comment:
This cost me hours!

Reply:
Sorry you had to go through this! 😭 

Thanks for figuring it out!!